pkg/kclient/kclient.go

```diff
@@ -49,3 +51,28 @@ func New() (*Client, error) {
 
 	return &client, nil
 }
+
+// CreateObjectMeta creates a common object meta
+func CreateObjectMeta(name, namespace string, labels, annotations map[string]string) metav1.ObjectMeta {
+
+	objectMeta := metav1.ObjectMeta{
+		Name:        name,
+		Namespace:   namespace,
+		Labels:      labels,
+		Annotations: annotations,
+	}
+
+	return objectMeta
+}
+
+// GetPVCsFromSelector returns the PVCs based on the given selector
+func (c *Client) GetPVCsFromSelector(selector string) ([]corev1.PersistentVolumeClaim, error) {
```

Comment:
Can this go in the volumes.go file? It'll make modularizing the kclient package easier in the future.

Reply:
Updated